test/diamorphosis/diamorphosis.test.ts

```diff
@@ -92,7 +92,7 @@ describe('Diamorphosis Test', () => {
             sasl: { mechanism: '', username: '', password: '' }
           },
           connectionTimeout: 1000,
```

Comment:
❓ Shall we bump connectionTimeout too to 3000?

Reply:
The default is 1000. I think we can do it on application level. Wdyt?